### PR TITLE
Implement serverinfo

### LINKED ISSUE #15: Public
- [x] ping
- [x] userinfo
- [x] serverinfo
- [x] avatar
- [x] banner

### PR DESCRIPTION
**Pull request by MaxGurbanli**
Anything else needed to be shown in the command?

### CODE REVIEW

File: cogs/public.py [hunk 1]
```diff
@@ -78,7 +78,31 @@ async def serverinfo(self, ctx: commands.Context):
         """
         Present server information.
         """
-        raise NotImplementedError('Command requires implementation and permission set-up.')
+        embed = discord.Embed(title=f"{ctx.guild.name} ({ctx.guild.id})", color=discord.Color.green())
+        creation_date = ctx.guild.created_at
+        # unix timestamp
+        timestamp = int(creation_date.timestamp())
+        embed.add_field(name="**Creation Date**", value=f"<t:{timestamp}> (<t:{timestamp}:R>)", inline=False)
```
Comment (ilikecubesnstuff) [inline cogs/public.py]: Probably good to use `discord.utils.format_dt` here (and if you are down for the task, replace existing uses of the project's `utils.time.format_timestamp` with `discord.utils.format_dt` and delete the former).

File: cogs/public.py [hunk 2]
```diff
@@ -78,7 +78,31 @@ async def serverinfo(self, ctx: commands.Context):
...
+        embed.add_field(name="**Verification Level**", value=f"{ctx.guild.verification_level}")
+        embed.add_field(name="**Owner**", value=f"{ctx.guild.owner}")
+
+        txt = len(ctx.guild.text_channels)
+        vc = len(ctx.guild.voice_channels)
+        ctg = len(ctx.guild.categories)
+        stg = len(ctx.guild.stage_channels)
+        frm = len(ctx.guild.forums)
+        total = txt + vc + ctg + stg + frm
+        embed.add_field(name="**Channels**", value=f"{txt} text, {vc} voice, {ctg} categories, {stg} stage, {frm} forum. {total} total", inline=False)
```
Comment (ilikecubesnstuff) [inline cogs/public.py]: Punctuation - `"[...] {total} total."`

**Review by ilikecubesnstuff — COMMENTED** [review]
Is the number of online users not displayed anywhere? It could be worth calling [`discord.Client.fetch_guild()`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.approximate_presence_count) with `with_counts=True` and then using [`Guild.approximate_member_count`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.approximate_member_count) and [`Guild.approximate_presence_count`](https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.approximate_presence_count) in the embed.

File: cogs/public.py (hunk 2 above)
Comment (MaxGurbanli) [reply cogs/public.py]: Hi, sorry I was on a grad trip for the last week. Requested changes arriving soon

File: cogs/reminder.py [hunk 3]
```diff
@@ -102,7 +103,7 @@ async def create(self, ctx: commands.Context, terminus: DatetimeConverter, *, me
         task = asyncio.create_task(self.handle_reminder(ctx.author, reminder))
         self.active[reminder.reminder_id] = task
         task.add_done_callback(self.removal_callback(reminder.reminder_id))
-        await ctx.send(f'Reminder set for {format_timestamp(terminus)} ({format_timestamp(terminus, "R")}).')
+        await ctx.send(f'Reminder set for {format_dt(terminus.timestamp())} ({format_dt(terminus.timestamp(), "R")}).')
```
Comment (ilikecubesnstuff) [inline cogs/reminder.py]: `format_dt` takes in a `datetime` object, which `terminus` is already. I don't think `datetime.timestamp` should be called here?

**Review by ilikecubesnstuff — APPROVED** [review]
LGTM!

**Review by NathanealV — APPROVED** [review]
LGTM!

**Pull request merged**